spec/daru/io/importers/redis_spec.rb

```diff
@@ -0,0 +1,66 @@
+RSpec.describe Daru::IO::Importers::Redis do
+	let(:connection) { Redis.new {} }
+	subject { Daru::IO::Importers::Redis.new(connection, *keys).load }
+
+  after { connection.flushdb }
```

Comment:
In daru we have rubocop disabled for specs because they are old. But in a new project, it is better to have it enabled.

Also take a look at `rubocop-rspec` gem, it adds several extremely useful checks related to RSpec logic.